src/components/CardPromo/CardPromo.tsx

```diff
@@ -0,0 +1,96 @@
+import { HTMLAttributes, useState } from "react";
+import styled from "styled-components";
+import { Icon, IconName, Text } from "@/components";
+
+export interface CardPromoProps extends HTMLAttributes<HTMLDivElement> {
+  label: string;
+  icon: IconName;
+  dismissible?: boolean;
+}
+const Background = styled.div`
+  ${({ theme }) => `
+  background-image: ${theme.click.card.promotion.color.stroke.default};
+  padding: 1px;
+  border-radius: ${theme.click.card.horizontal.radii.all};
+  box-shadow: ${theme.click.card.shadow};
+  display: flex;
+  `}
+`;
+const Wrapper = styled.div<{
+  $dismissible?: boolean;
+}>`
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: flex-start;
+  cursor: pointer;
+
+  ${({ theme }) => `
+    background: ${theme.click.card.promotion.color.background.default};
+    color: ${theme.click.card.promotion.color.text.default}; 
+    border-radius: ${theme.click.card.horizontal.radii.all};
```

Comment:
Should we use horizontal card here?
If we are using horizontal Card, Should we create a common wrapper for both?